### ISSUE TITLE
Ember's Flame Guard visual doesn't display damage taken after first spell

### ISSUE THREAD

**Issue by adezxc** [ValveSoftware/Dota2-Gameplay#10412]
### Ability name

Flame Guard

### Description

Ember's Shield meter visual only updates after one spell, the damage amount is calculated correctly, but the visual seems to be updated only after some damage. Works fine with other barriers (Tested Pipe, Pavice, Aphotic Shield)

https://github.com/ValveSoftware/Dota2-Gameplay/assets/16798732/c24c0acc-09eb-4fc1-a559-38d46465020e

### Example Match ID (and possibly Timestamp)

No Match ID

**Comment by adezxc** [comment]
Duplicate of #8884, closing
